pyproject.toml

```diff
@@ -10,6 +10,7 @@ load-plugins = ["pylint.extensions.bad_builtin", "pylint.extensions.mccabe"]
 extension-pkg-allow-list = ["yara"]
 jobs = 0
 include-naming-hint = true
+good-names = ["i", "j", "k", "ex", "Run", "_", "fo", "fw", "rv", "db", "id", "pw"]
```

Comment:
There are a couple more if you run `rg -g "*.py" 'as [[:alpha:]]{1,2}:'`
There are a couple we maybe don't want (tc? sc? fd?)
We probably also want "fp" though for the couple of places where we need to use `open()` and "id" is not a good name since it's a built-in function `id()` :sweat_smile:

Reply:
I'd say postpone the discussion about tc, sc and fd.
I added everything and removed id.